lib/nameToColor.js

```diff
@@ -8,9 +8,10 @@ const HEX = {
 
 const RGBA = {
   black: 'rgba(0, 0, 0, 0.75)',
-  red: 'rgba(219, 69, 0, 0.8)',
-  orange: 'rgba(245, 151, 0, 0.8)',
-  yellow: 'rgba(242, 202, 0, 0.7)',
+  blacklight: 'rgba(0, 0, 0, 0.5)',
```

Comment:
It was requested we lighten up the homepage hero translucency, and I did not want to change anything else that may have used the black color.